scripts/cddl_gen.py

```diff
@@ -969,6 +976,402 @@ def choice_var_access(self):
         return self.access_append(self.choice_var_name())
 
 
+class CddlValidationError(Exception):
+    pass
+
+# Subclass of tuple for holding key,value pairs. This is to make it possible to use isinstance() to
+# separate it from other tuples.
+class KeyTuple(tuple):
+    def __new__(cls, *in_tuple):
+        return super(KeyTuple, cls).__new__(cls, *in_tuple)
+
+class PythonClass(CddlXcoder):
```

Comment:
Should we have some docs on what this is used for?

Reply:
I means since we have docs for the 3 line class above :smile:

Reply:
Done :)